src/main/groovy/org/transmartproject/batch/clinical/ClinicalDataLoadJobConfiguration.groovy

```diff
@@ -267,20 +265,18 @@ class ClinicalDataLoadJobConfiguration extends AbstractJobConfiguration {
     }
 
     @Bean
-    ItemWriter<ClinicalFactsRowSet> factRowSetTableWriter() {
-        new ObservationFactTableWriter()
+    ItemWriter<PatientDimensionTableWriter> patientDimensionTableWriter() {
+        new PatientDimensionTableWriter()
```

Comment:
You're simultaneously declaring the bean and using `@Component`. Since the `batch.patient` package is being auto-loaded, you don't need this method.